ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/ManageSubscriptionsViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter.viewmodel
+
+import androidx.lifecycle.ViewModel
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.PurchaseInformation
+import com.revenuecat.purchases.ui.revenuecatui.data.PurchasesType
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+internal class ManageSubscriptionsViewModel(
```

Comment:
Hmm I'm wondering if we should have the VM at the screen level, or at the `CustomerCenter` level... That is, whether this should be `CustomerCenterViewModel`. We normally only display one screen at the same time, so it kinda makes sense to me to have the VM at that level. 

I guess having different VMs for different screens could allow us to separate logic more effectively, but IMO, most of the logic can be shared for all screens, wdyt?

Reply:
I like it! It will simplify the code tbh. Refactoring!!!